src/components/AqiGraph/AqiGraph.tsx

```diff
@@ -0,0 +1,311 @@
+import React, {useEffect, useState} from 'react';
+import {Heading, Flex} from '@chakra-ui/react';
+import {
+  ScatterChart,
+  XAxis,
+  YAxis,
+  Scatter,
+  CartesianGrid,
+  ResponsiveContainer,
+} from 'recharts';
+import firebase, {firestore} from '../../firebase';
+import {useTranslation} from 'react-i18next';
+
+/**
```

Comment:
Externalize the interfaces to another file like Util.ts or AqiGraph.types.ts or something